projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,41 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=harbor
+REPO_OWNER=goharbor
+REPO_SUBPATH=./src
+
+BASE_IMAGE_NAME?=eks-distro-base
+DOCKERFILE_FOLDER=./docker/linux/$(IMAGE_NAME)
+
+FETCH_BINARIES_TARGETS=eksa/helm/chartmuseum
+
+IMAGE_NAMES=harbor-chartserver
+
+HARBOR_CHARTSERVER_IMAGE_COMPONENT=harbor/harbor-chartserver
```

Comment:
I think here just set IMAGE_COMPONENT

Reply:
That is right if it has only one image. I name it that way because we are expecting more image components to be added.